modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -21,6 +21,58 @@ definition {
 		}
 	}
 
+	@description = "LPS-161924 - Verify that it's possible to add attachments to the notification template"
+	@priority = "3"
+	test CanAddAttachment {
+		task ("Creating a object and a attachment field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161924",
+				objectName = "CustomObject161924",
+				pluralLabelName = "Custom Objects 161924");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Attachment",
+				fieldLabelName = "From Computer And Show Files In DM",
+				fieldName = "fromComputerAndShowFilesInDM",
+				fieldType = "Long",
+				fileSource = "userComputer",
+				isRequired = "false",
+				objectName = "CustomObject161924");
+		}
+
+		task ("Opening the templates and adding infos") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				fromAddress = "test@liferay",
+				fromName = "Test Test",
+				name = "Notification Template");
+		}
+
+		task ("Adding the attachment and saving") {
+			ObjectNotifications.addNotificationTemplateAttachment(objectLabel = "Custom Object 161924");
+
+			Button.clickSave();
+		}
+
+		task ("Entering in the template and asserting that the attachment is present") {
+			Click(
+				key_objectLabel = "Notification Template",
+				locator1 = "ObjectAdmin#OBJECT_KEBAB_MENU");
+
+			MenuItem.clickNoError(menuItem = "View");
+
+			AssertElementPresent(
+				key_dataSource = "Custom Object 161924",
+				locator1 = "ObjectNotifications#DATA_SOURCE_ATTACHMENTS");
+
+			AssertElementPresent(locator1 = "ObjectNotifications#FIELD_ATTACHMENTS_OBJECT");
+		}
```

Comment:
Change the task to: `Then the attachment should be present in the notification template`